samples/opensource/trtexec/trtexec.cpp

```diff
@@ -798,6 +816,17 @@ int main(int argc, char** argv)
 
     initLibNvInferPlugins(&gLogger.getTRTLogger(), "");
 
+#if !defined(_WIN32)
+    for (auto plugin : gParams.plugins)
+    {
+        void *dlhandle = dlopen(plugin.c_str(), RTLD_LAZY);
```

Comment:
We should create a cross-platform helper in `common.h` for loading libraries. I think the Windows equivalent is `LoadLibrary`

Reply:
Do we support windows platform in OSS build yet?

Reply:
fixed